rpm/mosh.spec

```diff
@@ -1,21 +1,22 @@
 Name:		mosh
-Version:	1.2.5
+Version:	1.3.2
 Release:	1%{?dist}
 Summary:	Mobile shell that supports roaming and intelligent local echo
 
 License:	GPLv3+
 Group:		Applications/Internet
 URL:		https://mosh.org/
-Source0:	https://github.com/downloads/keithw/mosh/mosh-%{version}.tar.gz
+%undefine       _disable_source_fetch
+Source0:	https://github.com/mobile-shell/mosh/archive/refs/tags/%{name}-%{version}.tar.gz
```

Comment:
This should use the release tarball (https://github.com/mobile-shell/mosh/releases/download/mosh-1.3.2/mosh-1.3.2.tar.gz), not the raw archive tarball. This also explains why were seeing a duplicated top-level directory name name, and why you needed to rerun `autogen.sh`.